libraries/AP_Filesystem/AP_Filesystem_Param.cpp

```diff
@@ -470,7 +470,7 @@ int AP_Filesystem_Param::stat(const char *name, struct stat *stbuf)
     }
     memset(stbuf, 0, sizeof(*stbuf));
     // give fixed size to avoid needing to scan entire file
```

Comment:
comment should be adjusted

Reply:
I changed the comment, squashed and rebased.